src/main/java/org/jcodec/containers/mp4/MP4Util.java

```diff
@@ -155,7 +179,7 @@ public void copy(SeekableByteChannel input, WritableByteChannel out) throws IOEx
         }
     }
 
-    public static MovieBox parseMovie(File source) throws IOException {
+    public static Movie parseMovie(File source) throws IOException {
```

Comment:
it's an API incompatible change 
i recommend creating a separate method e.g. `movieFromFile` and use it instead of changing public method signature